### PR TITLE
fs: add sed support

### PR DESCRIPTION
**Pull request by aborah-sudo**
Add sed functionality to new test framework

### CODE REVIEW

File: pytest_mh/utils/fs.py [hunk 1]
```diff
@@ -738,3 +738,22 @@ def chown(
...
+        :type command: str
+        :param path: File where changes will happen
+        :type path: str
+        :param args: Additional options, defaults to None
+        :type args: list[str] | None, optional
+        :return: Result of process
+        :rtype: SSHProcessResult
+        """
+        self.backup(path)
+        self.logger.info(f'Changing file "{path}"')
```
Comment (pbrezina) [inline pytest_mh/utils/fs.py:757]: Hi, can you log full sed command here? That would be more helpful in the logs. E.g.:

`Running sed s/a/b/g on /etc/path`

Comment (aborah-sudo) [reply pytest_mh/utils/fs.py]: Done

**Comment by jakub-vavra-cz** [comment]
@aborah-sudo Looks like You need to run the code through black to fix the ci.

**Comment by aborah-sudo** [comment]
> @aborah-sudo Looks like You need to run the code through black to fix the ci.

Done

**Comment by aborah-sudo** [comment]
> Hi, thank you. It looks good.
> 
> Before we push it, can you just rename the commit subject to `fs: add sed support` to follow the convention of other commits?

Done

**Review by pbrezina — APPROVED** [review]

**Pull request merged**